common/lib/xmodule/xmodule/course_module.py

```diff
@@ -855,6 +855,14 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    minimum_grade_credit = Float(
```

Comment:
A couple of comments--

It was my understanding that enabling a course for credit will not be in Advanced Settings, because we wish to limit that to global staff (vs. course staff). Given that, it seems odd to show an Advanced Setting about Minimum Grade for Credit to all Studio users. Can't this also be set in the same way that enabling a course for credit is set? I think that @wedaly and @ormsbee had settled on a way to do this outside of Studio.

If this does stay in Advanced Settings, I suggest improving the help message so it is clearer what 0 and 1.0 correspond to (0% and 100%?).